cloudbuild.yaml

```diff
@@ -0,0 +1,12 @@
+steps:
+- name: 'gcr.io/cloud-builders/docker'
+  entrypoint: 'bash'
+  args:
+  - '-c'
+  - |
+    docker build --build-arg crystal_deb=crystal_0.27.0-1_amd64.deb -t "gcr.io/$PROJECT_ID/trusty_crystal:$COMMIT_SHA" .
+    if [ "$BRANCH_NAME" == "master" ]; then
```

Comment:
If you use the `[[` operator, you don't need to quote the arguments. From TFM:
> Word splitting and pathname expansion are  not  performed on  the words between the [[ and ]]; tilde expansion, parameter and variable expansion, arithmetic expansion, command substitution, process substitution, and quote removal are performed.

Ah, shell.

Reply:
Nice 👍